casepro/backend/tests/test_junebug.py

```diff
@@ -402,6 +579,55 @@ def request_callback(request):
         self.assertEqual(sorted(res), sorted(['+1234', '+4321']))
 
     @responses.activate
+    def test_get_identities(self):
+        '''The get_identities function should call the correct URL, and return
+        the relevant identities.'''
+        identity_store = IdentityStore(
+            'http://identitystore.org/', 'auth-token', 'msisdn')
+
+        def request_callback(request):
```

Comment:
Could we put this callback in a separate method like you did with the others? That looked really clean.